generator/modules/python.py

```diff
@@ -18,6 +18,7 @@ def build(self):
             DEBIAN_FRONTEND=noninteractive $APT_INSTALL \
                 python3-pip \
                 python3-dev \
+                python3-distutils \
```

Comment:
Seems not working for Ubuntu 16.04.
Could you fix this issue?

Reply:
@ufoym Thanks for the review! I've improved this PR for `distutils` issue in Ubuntu versions.

Tested for Ubuntu 16.04 and 18.04.
Should be good now, could you verify again? Thanks!